photon-server/src/main/java/org/photonvision/server/CameraSocketHandler.java

```diff
@@ -56,16 +58,17 @@ private CameraSocketHandler() {
     }
 
     public void onConnect(WsConnectContext context) {
-        context.session.setIdleTimeout(Long.MAX_VALUE); // TODO: determine better value
-        var insa = context.session.getRemote().getInetSocketAddress();
-        var host = insa.getAddress().toString() + ":" + insa.getPort();
+        context.session.setIdleTimeout(
+                Duration.ofMillis(Long.MAX_VALUE)); // TODO: determine better value
```

Comment:
feels like a good time to find a better value

Reply:
The frontend doesn't even connect to the CameraWebsocket anymore. What from this system is even required anymore?

Reply:
Yeah kill the references here I think

Reply:
There are a lot of requests and things in photon-server which aren't used anymore. Like that entire UISettings class